src/components/Cover/index.js

```diff
@@ -17,137 +16,154 @@ const Container = styled.div`
   z-index: 4;
   background-color: ${color.space};
   width: 100%;
-  height: ${props => (props.height === 'full' ? '100vh' : 'auto')};
+  height: ${props => (props.fullHeight ? '100vh' : 'auto')};
   color: white;
+  padding: ${space[16]};
 `
 
 const BackgroundImage = styled.div`
+  filter: ${props => props.blurred && 'blur(15px)'};
```

Comment:
When not blurred this would leave an empty value. We should return css when blurred is true, otherwise, nothing.